Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,44 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses the interface defined `pwd.h` to add a new user to the system
+password file. 
+
+A user can either be a `login` user (that is, they have a home
+directory; someone who actually uses to system) or a `system` user (such
+as the `lookup` user found in `/etc/passwd`). 
+
+By default, the user will be added to the `users` group, which has a gid
+of 100.
+
+This program must be run as root.
+
+## Options
+
+* `-u` _UID_: The User Idenfitier for the new user. If this is not specified, a value starting from `1000` will be specified.
+* `-g` _GID_: The Group Identifier for the new user. If this is not specified, it will default to 100 (the Serenity **user** group).
+* `-s` _SHELLBIN_: The shell binary for this login. The default is `/bin/sh`. 
+* `-m`: Create a home directory for this new user at `/home/username>`.
+* `-d` _HOMEPATH_: Set the home directory for this user to _HOMEPATH).
+* `-n` _GECOS_: GECOS information about this login. See [here](https://en.wikipedia.org/wiki/Gecos_field) for more information.
+
+## Examples
+
+```sh
+# su
```

Comment:
No need to show `su` here. Also the `#` indicates that we're already `root`. :)

Reply:
Ahh I see!